### PR TITLE
Events list page

### PR DESCRIPTION
**Pull request by feedmypixel**
I wanted to put this up for discussion around points 2 and 3 below. Once done I can get on with the tests

- [x] Events link in navigation
- [x] Events listing page
- [x] Tests (only unit - will do acceptance tests once #631 is in)

## Of Note
Sorting is reliant on https://github.com/uktrade/data-hub-leeloo/pull/478

screencapture-localhost-3000-events-1505896734884

### CODE REVIEW

File: src/apps/events/router.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
 const router = require('express').Router()
 
+const { setDefaultQuery } = require('../middleware')
+const { getRequestBody, getEventsCollection } = require('./middleware/collection')
+const { renderEventList } = require('./controllers/list')
+
 const {
   details,
```
Comment (feedmypixel) [inline src/apps/events/router.js]: @yeahfro as mentioned I feel we may be loosing some clarity here using `index.js` files. I quite like the approach taken with variable names a good example can be seen in [contacts router](https://github.com/uktrade/data-hub-frontend/blob/develop/src/apps/contacts/router.js)

**Comment by ztolley** [comment]
Also needs an add button

File: src/apps/events/controllers/list.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const { get, omit, merge } = require('lodash')
+const { eventFiltersFields: filtersFields, eventSortForm } = require('../macros')
+const { buildSelectedFiltersSummary } = require('../../builders')
+
+function renderEventList (req, res) {
+  const sortForm = merge({}, eventSortForm, {
+    hiddenFields: Object.assign({}, omit(req.query, 'sortby')),
```
Comment (yeahfro) [inline src/apps/events/controllers/list.js]: How about `assign` over `Object.assign`?

File: src/apps/events/labels.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const labels = {
+  collectionFilterLabels: {
```
[the remark this answers is not in the document]
Comment (yeahfro) [reply src/apps/events/labels.js]: Thank you!

File: test/unit/data/events/event-data.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
+{
```
Comment (feedmypixel) [inline test/unit/data/events/event-data.json]: @yeahfro FYI I have dropped in a fuller `event-data.json` mock. I didn't want to remove the current [event.json](https://github.com/uktrade/data-hub-frontend/blob/develop/test/unit/data/events/event.json) as it shouldn't be part of this PR. But it feels like maybe we should just have one?

File: src/apps/events/transformers.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  address_1,
+  address_2,
+  address_town,
+  address_county,
+  address_postcode,
+  notes,
+  organiser,
+  lead_team,
+  uk_region,
+} = {}) {
```
Comment (tyom) [inline src/apps/events/transformers.js]: I think this is unnecessary. We expect the first argument to be an object.
